### PR TITLE
[devtools-translate] Korean - What's New in DevTools (Chrome 103)

### LINKED ISSUE #3014: [devtools-translate] Japanese - What's New in DevTools (Chrome 103)
Post: https://developer.chrome.com/blog/new-in-devtools-103/
Translated by: June 19

Translator: @yoichiro
Reviewers: @yoshiko-pg, @lacolaco, @technohippy

### LINKED ISSUE #3015: [devtools-translate] Korean - What's New in DevTools (Chrome 103)
Post: https://developer.chrome.com/blog/new-in-devtools-103/
Translated by: June 19

Translator: @techhtml
Reviewers: @cwdoh @TORU0239

### PR DESCRIPTION
**Pull request by techhtml**
Fixes #3015 
- [Translate What's new in devtools 103 into Korean](https://github.com/GoogleChrome/developer.chrome.com/commit/3166aa33630af9cf074c452e6303fc1c37c18903)
- [translate whats-new.md into Korean](https://github.com/GoogleChrome/developer.chrome.com/commit/e0d42977737df93ce40ba578689d28f82a6cc796)

### CODE REVIEW

**Comment by jecfish** [comment]
Thanks Eun Cho! Over to @cwdoh for review

File: site/ko/blog/new-in-devtools-103/index.md [hunk 1]
```diff
@@ -5,173 +5,164 @@ authors:
...
-<!-- - **Double-click** is captured as `type: doubleClick`.
-- **Right-click** event is captured as `type: click` but with the `button` property is set to `secondary`. The `button` value of a normal mouse click is `primary`. -->
+- **더블 클릭** 은 `type: doubleClick` 으로 캡쳐됩니다.
+- **우 클릭** 이벤트는 `type: click` 으로 캡쳐되지만 `button` 속성은 `secondary` 로 설정됩니다. 일반적인 마우스 클릭의 `button` 값은 `primary` 입니다.
 
 Chromium issues: [1300839](https://crbug.com/1300839), [1322879](https://crbug.com/1322879), [1299701](https://crbug.com/1299701), [1323688](https://crbug.com/1323688)
 
 
-<!-- ## New timespan and snapshot mode in the Lighthouse panel {: #lighthouse } -->
+## Lighthouse 패널에서 새로 출시된 timspan 과 snapshot 모드 {: #lighthouse }
```
Comment (cwdoh) [inline site/ko/blog/new-in-devtools-103/index.md:42]: * timspan -> timespan
* remove a space right before 과

File: site/ko/blog/new-in-devtools-103/index.md [hunk 2]
```diff
@@ -5,173 +5,164 @@ authors:
...
 
 
-<!-- ## Performance Insights updates {: #performance } -->
+## Performance Insights 업데이트 {: #performance }
 
-<!-- ### Improved zoom control in the Performance Insights panel {: #zoom } -->
+### Performance Insights 패널에서 zoom 컨트롤 개선 {: #zoom }
 
-<!-- DevTools will now zoom in based on your mouse cursor rather than the playhead position.With the latest cursor-based zoom, you can move your mouse to anywhere in the track, and [zoom in](/docs/devtools/performance-insights/#navigate) to the desired area right away.  -->
+DevTools 는 이제 재생 헤드 위치가 아닌 마우스 커서를 기반으로 줌됩니다. 커서를 중심으로 한 줌을 사용하면 트랙 내 어디에나 마우스를 옮겨 원하는 곳으로 [줌 인](/docs/devtools/performance-insights/#navigate) 할 수 있습니다.
```
Comment (cwdoh) [inline site/ko/blog/new-in-devtools-103/index.md:69]: How about translating `latest cursor-based zoom` to just '커서 기반 줌'???

File: site/ko/blog/new-in-devtools-103/index.md [hunk 3]
```diff
@@ -5,173 +5,164 @@ authors:
...
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/XMHyRsyK24fWLK7o72K7.png", alt="Improved inline value preview during debugging", width="800", height="534" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/XMHyRsyK24fWLK7o72K7.png", alt="디버깅 중 인라인 값 미리보기 개선", width="800", height="534" %}
 
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/8e1a99324bde8d093e32ede5c8d1bf50110fac66 #}
 
 Chromium issue: [1316340](https://crbug.com/1316340) 
 
 
-<!-- ## Support large blobs for virtual authenticators {: #webauthn } -->
+## 가상 인증자를 위한 대규모 Blob 지원 {: #webauthn }
```
Comment (cwdoh) [inline site/ko/blog/new-in-devtools-103/index.md:126]: 대규모 -> 대용량 or 큰 사이즈의

**Comment by techhtml** [comment]
@jecfish I applied all reviews, Feel free to review again :)

**Review by jecfish — APPROVED** [review]

**Comment by jecfish** [comment]
Thank you all!

**Pull request merged**